### PR TITLE
[15.0][MIG] sale_stock_operating_unit: Migration to 15.0

### PR DESCRIPTION
**Pull request by BT-apadron**

### CODE REVIEW

File: sale_stock_operating_unit/test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo-addon-sale_operating_unit @ git+https://github.com/OCA/operating-unit/pull/479/head#subdirectory=setup/sale_operating_unit
```
Comment (Saran440) [inline sale_stock_operating_unit/test-requirements.txt:1]: Is this file still used? I think it can delete

**Review by Saran440 — COMMENTED** [review]
@BT-apadron Can you update your code please?

**Comment by AaronHForgeFlow** [comment]
Hi all, I am continuing here: https://github.com/OCA/operating-unit/pull/594 as long there is some time since the last update here. Just let me know if you prefer to continue on this one.

**Pull request closed without merging**